src/main/java/frc/robot/subsystems/intake/IntakeSubsystem.java

```diff
@@ -66,9 +72,13 @@ public void toggle() {
     }
 
     private void idle() {
-        if (action == IntakeAction.INTAKE) {
+        if (action == IntakeAction.INTAKE
+                && (!inputs.noteSensed
+                        && !noteInShooterDeck
+                                .getAsBoolean())) { // dont seek if note in uptake or shotoer deck
             state = State.SEEKING;
-        } else if (action == IntakeAction.REVERSE) {
+        } else if (action == IntakeAction.REVERSE
```

Comment:
Don't we only want to auto reverse if we have 2 notes?

Reply:
yeah i forgot to carry that over from other PR, pushing commit now